### PR TITLE
No more need for SANDBOX_ENABLE_ECOMMERCE

### PR DESCRIPTION
**Pull request by nedbat**
Configuration Pull Request
---

Make sure that the following steps are done before merging:

  - [ ] A DevOps team member has approved the PR.
  - [ ] Are you adding any new default values that need to be overridden when this change goes live? If so:
    - [ ] Update the appropriate internal repo (be sure to update for all our environments)
    - [ ] If you are updating a secure value rather than an internal one, file a DEVOPS ticket with details.
    - [ ] Add an entry to the CHANGELOG.
  - [ ] Have you performed the proper testing specified on the [Ops Ansible Testing Checklist](https://openedx.atlassian.net/wiki/display/EdxOps/Ops+Ansible+Testing+Checklist)?

### CODE REVIEW

**Comment by nedbat** [comment]
@edx/devops Review?  This is already in Ginkgo.

File: playbooks/edx_sandbox.yml [hunk 1]
```diff
@@ -21,10 +21,6 @@
     edx_platform_version: 'master'
     # Set to false if deployed behind another proxy/load balancer.
     NGINX_SET_X_FORWARDED_HEADERS: True
-    # These should stay false for the public AMI
-    COMMON_ENABLE_DATADOG: False
-    SANDBOX_ENABLE_ECOMMERCE: False
-    COMMON_ENABLE_SPLUNKFORWARDER: False
```
Comment (jibsheet) [inline playbooks/edx_sandbox.yml:27]: I assume these are defaulted to false, so you're not repeating that falseness?

**Review by jibsheet — APPROVED** [review]
Assuming my question about the COMMON_ENABLE* vars makes sense, the rest looks fine

File: playbooks/edx_sandbox.yml (hunk 1 above)
Comment (nedbat) [reply playbooks/edx_sandbox.yml:27]: COMMON_ENABLE_DATADOG and COMMON_ENABLE_SPLUNKFORWARDER are defaulted to False in playbooks/roles/common_vars/defaults/main.yml.  SANDBOX_ENABLE_ECOMMERCE is completely removed by this pull request.

**Pull request merged**